app/page.tsx

```diff
@@ -1,101 +1,29 @@
-import Image from "next/image";
+import Button from "@/app/components/button";
+import { EnvelopeIcon } from "@heroicons/react/20/solid";
```

Comment:
LOOKS GREAT SO FAR - can we match the design a bit more by changing this to be the outline version of the icon, instead of solid?
```suggestion
import { EnvelopeIcon } from "@heroicons/react/20/outline";
```
if there was a specific reason for using solid icons tho, nw! just document why :-)